.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+on:
```

Comment:
Can you name workflows? Make the checks look nicer?

Reply:
Done, I just went with CI to be concise